features/frontend/checkout_security.feature

```diff
@@ -41,27 +41,27 @@ Feature: Checkout security
     Scenario: Signing in during the checkout
          When I fill in the following:
             | Email    | john@example.com |
-            | Password | foo              |
+            | Password | foo1             |
           And I press "Login"
          Then I should be redirected to the checkout addressing step
 
     Scenario: Creating account during the checkout
          When I fill in the following:
-            | fos_user_registration_form_email                | mike@example.com |
-            | fos_user_registration_form_plainPassword_first  | mikepass         |
-            | fos_user_registration_form_plainPassword_second | mikepass         |
-            | fos_user_registration_form_firstName            | Mike             |
-            | fos_user_registration_form_lastName             | Small            |
+            | sylius_customer_registration_firstName                 | Mike             |
+            | sylius_customer_registration_lastName                  | Small            |
+            | sylius_customer_registration_email                     | mike@example.com |
+            | sylius_customer_registration_user_plainPassword_first  | mikepass         |
+            | sylius_customer_registration_user_plainPassword_second | mikepass         |
           And I press "Register"
          Then I should be redirected to the checkout addressing step
 
     Scenario: Creating account during the whole checkout
          When I fill in the following:
-            | fos_user_registration_form_email                | mike@example.com |
-            | fos_user_registration_form_plainPassword_first  | mikepass         |
-            | fos_user_registration_form_plainPassword_second | mikepass         |
-            | fos_user_registration_form_firstName            | Mike             |
-            | fos_user_registration_form_lastName             | Small            |
+            | sylius_customer_registration_firstName                 | Mike             |
+            | sylius_customer_registration_lastName                  | Small            |
+            | sylius_customer_registration_email                     | mike@example.com |
+            | sylius_customer_registration_user_plainPassword_first  | mikepass         |
+            | sylius_customer_registration_user_plainPassword_second | mikepass         |
```

Comment:
We have Registration, Login and Guest form rendered on the same page. Labels are not unique.